Engines/Wine/QuickScript/QuickScript/script.js

```diff
@@ -42,6 +49,21 @@ QuickScript.prototype.category = function(category) {
 };
 
 /**
+ * set miniature (for the installation and the shortcut)
+ * @param miniature path to the miniature file
+ */
+QuickScript.prototype.miniature = function(miniature) {
+    // get
+    if (arguments.length == 0) {
+        return this._miniature;
+    }
+
+    // set
+    this._miniature = java.util.Optional.of(miniature);
```

Comment:
Is it ensured at this point that `miniature` is not `null`?

Reply:
No. But I think it is ok to get an exception if you call `miniature` with `null` as argument.